frontendFile/src/main/java/com/virtuslab/gitmachete/frontend/file/highlighting/MacheteAnnotator.java

```diff
@@ -28,16 +37,30 @@ public void annotate(PsiElement element, AnnotationHolder holder) {
     }
   }
 
+  @UIEffect
+  private void showCantGetBranchesMessage(PsiFile file) {
+    Editor currentEditor = FileEditorManager.getInstance(file.getProject()).getSelectedTextEditor();
+    if (currentEditor == null) {
+      return;
+    }
+    HintManager.getInstance().showErrorHint(currentEditor,
+        "We can't get project branches, so we can't checking it names", HintManager.ABOVE);
```

Comment:
Maybe just:
```suggestion
        "Cannot get git branches", HintManager.ABOVE);
```

Reply:
As agreed via Slack, we leave it as is for clarity

Reply:
Just fix the wording/grammar:
```suggestion
        "We can't get project branches, so we can't complete their names", HintManager.ABOVE);
```

Reply:
done on following PR